base/strings/basic.sublime-workspace

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
This file seems like it got accidentally added.

Reply:
Yeah! :sweat_smile: